recipes/so3/all/conanfile.py

```diff
@@ -0,0 +1,58 @@
+from conans import CMake, ConanFile, tools
+from conans.errors import ConanInvalidConfiguration
+from glob import glob
+import os
+
+
+
+class So3Conan(ConanFile):
+    name = "so3"
+    license = "GPL-3.0-or-later"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/astro-informatics/so3"
+    description = "Fast and accurate Wigner transforms"
+    settings = "os", "arch", "compiler", "build_type"
+    topics = ("physics", "astrophysics", "radio interferometry")
+    options = {"fPIC": [True, False]}
+    default_options = {"fPIC": True}
+    requires = "fftw/3.3.9", "ssht/1.3.7"
+    generators = "cmake", "cmake_find_package"
+    exports_sources = ["CMakeLists.txt"]
+
+    @property
+    def _source_subfolder(self):
+        return "source_subfolder"
+
+    @property
+    def _build_subfolder(self):
+        return "build_subfolder"
+
+    def config_options(self):
+        if self.settings.compiler == "Visual Studio":
+            raise ConanInvalidConfiguration("SO3 requires C99 support for complex numbers.")
+        del self.settings.compiler.cppstd
+        del self.settings.compiler.libcxx
+
+    def source(self):
+        tools.get(**self.conan_data["sources"][self.version])
+        extracted_dir = glob('so3-*/')[0]
+        os.rename(extracted_dir, self._source_subfolder)
+
+    @property
+    def cmake(self):
+        if not hasattr(self, "_cmake"):
+            self._cmake = CMake(self)
+            self._cmake.definitions["tests"] = False
```

Comment:
```suggestion
            self._cmake.definitions["tests"] = False
```

@mdavezac I see you are so3 author or contributor at least. Since CMake 3.15 there is a native flag [BUILD_TESTING](https://cmake.org/cmake/help/v3.15/command/enable_testing.html), which is much better.

Reply:
Good to know! Got rid of the homegrown "tests" option in the original CMake.